app-frontend/packs/components/dashboard/homePage/HomePageBanner.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+
+    <div
+        class="tt-home-page-banner">
```

Comment:
и поправь тут мультилайн и ниже

P.S. но если используется только один атрибут, мультилайн можно не использовать)
```suggestion
    <div
        class="tt-home-page-banner"
    >
```